yb-voyager/src/query/queryissue/parser_issue_detector.go

```diff
@@ -305,16 +305,17 @@ func (p *ParserIssueDetector) getOrCreateTableMetadata(tableName string) *TableM
 		schemaName = parts[0]
 		tableNameOnly = parts[1]
 	}
-	usageCategory := p.getUsageCategoryForTable(schemaName, tableNameOnly)
 	tm := &TableMetadata{
 		TableName:   tableNameOnly,
 		SchemaName:  schemaName,
-		Usage:       usageCategory,
+		Usage:       ObjectUsageCategoryUnused, //start with  unused and populating it in FinalizeColumnMetadata->buildUsageCategoryForAllTables()
 		Columns:     make(map[string]*ColumnMetadata),
 		Constraints: make([]ConstraintMetadata, 0),
 		Indexes:     make([]*queryparser.Index, 0),
 	}
+	tm.Usage = p.getUsageCategoryForTable(tm)
```

Comment:
as discussed, let's remove this for now as plpgsql having "create table" is not very likely.